common/util/src/main/java/org/eclipse/dataspaceconnector/common/annotations/ExcludeFromDependencyAnalysisReport.java

```diff
@@ -0,0 +1,30 @@
+/*
+ *  Copyright (c) 2022 Microsoft Corporation
+ *
+ *  This program and the accompanying materials are made available under the
+ *  terms of the Apache License, Version 2.0 which is available at
+ *  https://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  SPDX-License-Identifier: Apache-2.0
+ *
+ *  Contributors:
+ *       Microsoft Corporation - initial API and implementation
+ *
+ */
+
+package org.eclipse.dataspaceconnector.common.annotations;
+
+import java.lang.annotation.ElementType;
+import java.lang.annotation.Retention;
+import java.lang.annotation.RetentionPolicy;
+import java.lang.annotation.Target;
+
+/**
+ * Exclude a class from dependency analysis.
+ *
+ * @see <a href="https://github.com/autonomousapps/dependency-analysis-android-gradle-plugin">Gradle Dependency Analysis Plugin</a>
+ */
+@Retention(RetentionPolicy.RUNTIME)
+@Target({ElementType.METHOD, ElementType.TYPE})
```

Comment:
Does it make sense to have here also ElementType.METHOD?

Reply:
Annotation removed as it did not accomplish the goal is was meant for